webapp/commonComponents/errorBadge.scss

```diff
@@ -14,4 +18,13 @@
     font-weight: bold;
     color: $red;
   }
+
+  &.warning {
+    background-color: transparent;
```

Comment:
isn't is inherited from ```error-badge-inverse```?

Reply:
apparently the definition .error-badge.warning is more specific than .error-badge-inverse.warning and overrides it